charts/osm/values.schema.json

```diff
@@ -702,6 +702,23 @@
                     },
                     "additionalProperties": false
                 },
+                "multicluster": {
+                    "$id": "#/properties/OpenServiceMesh/properties/multicluster",
+                    "type": "object",
+                    "title": "Multicluster",
+                    "description": "Configuration for multicluster features",
+                    "required": [
+                        "gatewayPort"
+                    ],
+                    "properties": {
+                        "gatewayPort": {
+                            "$id": "#/properties/OpenServiceMesh/properties/multicluster/properties/gatewayPort",
+                            "type": "integer",
+                            "title": "Gateway port",
+                            "description": "Port number of multicluster gateway"
```

Comment:
not a must but most of the other objects in this schema also include an `example` category. May be good to add it for uniformity?

Reply:
Resolved